integration/commitlog_bootstrap_test.go

```diff
@@ -82,7 +82,24 @@ func TestCommitLogBootstrap(t *testing.T) {
 	seriesMaps := generateSeriesMaps(30, now.Add(-2*blockSize), now.Add(-blockSize))
 	log.Info("writing data")
 	writeCommitLogData(t, setup, seriesMaps, testNamespaces[0])
-	log.Info("written data")
+	log.Info("finished writing data")
+
+	// The datapoints in this generated data are within the retention period and
+	// would oridinarly be bootstrapped, however, we intentionally write them to a
+	// commitlog file that has a timestamp outside of the retention period. This
+	// allows us to verify the commitlog bootstrapping logic will not waste time
+	// reading commitlog files that are outside of the retention period.
+	log.Info("generating data")
+	seriesMapsExpiredCommitlog := generateSeriesMaps(30, now.Add(-2*blockSize), now.Add(-blockSize))
+	log.Info("writing data to commitlog file with out of range timestamp")
```

Comment:
Might be better to add this as a separate independent test so we can tell what part of the commit log is failing based on test name, rather than add this to a even larger compound integration test.

Reply:
Yeah fair enough, I just did it because these tests can take awhile to run and this seemed like a fairly small feature to test in an integration test, but I can break it out